src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/Recipes/ElasticIndexResetStep.cs

```diff
@@ -28,31 +28,32 @@ public async Task ExecuteAsync(RecipeExecutionContext context)
             {
                 await HttpBackgroundJob.ExecuteAfterEndOfRequestAsync("elastic-index-reset", async scope =>
                 {
-
                     var elasticIndexingService = scope.ServiceProvider.GetService<ElasticIndexingService>();
                     var elasticIndexSettingsService = scope.ServiceProvider.GetService<ElasticIndexSettingsService>();
                     var elasticIndexManager = scope.ServiceProvider.GetRequiredService<ElasticIndexManager>();
 
-                    var indices = model.IncludeAll ? (await elasticIndexSettingsService.GetSettingsAsync()).Select(x => x.IndexName).ToArray() : model.Indices;
+                    var indexNames = model.IncludeAll ? (await elasticIndexSettingsService.GetSettingsAsync()).Select(x => x.IndexName).ToArray() : model.Indices;
```

Comment:
if you rename this then to be consistent you need to change it for model.Indices too?